docs/readmes/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -74,3 +78,80 @@ hide_title: true
     ```bash
     apt show magma
     ```
+
+## Steps with Bazel
+
+This is not supported for 1.8, but is possible on the development branch and will replace the documentation above in future versions.
+
+1. **Clone or Update  your local repository**.
+    In any machine (i.e. your computer) clone the magma repo.
+
+    ```bash
+    git clone https://github.com/magma/magma.git
+    ```
+
+2. **Install prerequisites**.
+    Make sure you have installed all the tools specified in the prerequisites <https://magma.github.io/magma/docs/basics/prerequisites#prerequisites>
```

Comment:
For now, the official documentation is still at https://docs.magmacore.org/, even though the deployment to that documentation is broken atm.
It is true, that the Github pages version is up to date and we should discuss if we want to switch to it as main documentation reference.
However, until a decision is reached regarding the deployment infra of the docs, I would keep it consistent and use the above mentioned reference.